app.py

```
@@ -323,7 +338,9 @@ async def test_evaluate_text(request: TextEvaluationRequest, pronunciation_score
         if "main_response" in _next:
             if "conversation" in _next.get("main_response"):
                 print("next_npc_ :", _next["main_response"]["conversation"][-1]["npc"])
-
+                next_npc_conversation = _next["main_response"]["conversation"][-1]["npc"]
+                tts_file_name = await convert_text_to_speech(next_npc_conversation)
```

Ruff (0.8.2)

Local variable `tts_file_name` is assigned to but never used

Remove assignment to unused variable `tts_file_name`

(F841)